mlflow/R/mlflow/R/tracking-rest.R

```diff
@@ -60,37 +60,57 @@ mlflow_rest <- function( ..., client, query = NULL, data = NULL, verb = "GET", v
   get_response <- switch(
     verb,
     GET = function() {
-      GET( api_url, query = query, mlflow_rest_timeout(), config = rest_config$config,
-           req_headers)
+     GET(
+       api_url,
+       query = query,
+       mlflow_rest_timeout(),
+       config = rest_config$config,
+       req_headers
+     )
```

Comment:
Proposed drive-by fixing these. Honestly, I'm not sure why these were implemented how they were in the first place